packages/editor/src/components/post-featured-image/index.js

```diff
@@ -230,6 +271,12 @@ function PostFeaturedImage( {
 												onRemoveImage();
 												toggleRef.current.focus();
```

Comment:
This line triggers an error when trying to remove the missing featured image because the toggle button is no longer rendered. We'll need to fix the focus return on removal logic.

## Screenshot
![CleanShot 2024-11-14 at 17 52 52](https://github.com/user-attachments/assets/6cf1d16e-b666-4aa8-aa2d-a8108c93b6e3)

Reply:
Good catch 👍🏻